pgjdbc/src/main/java/org/postgresql/PGProperty.java

```diff
@@ -448,6 +448,17 @@
       false,
       new String[] {"3"}),
 
+  /**
+   * Quote returning columns.
+   * There are some ORM's that quote everything, including returning columns
+   * If we quote them, then we end up with a column that looks like ""colname""
```

Comment:
I'm unclear here. why would there be 3 quotes ?

Reply:
If the quoting code is triggered it will always start and end with a quote and between these quotes will add the quoted string. So for input `"column"` you end up with `" + ""column"" + "`

Reply:
OK, I think I understand, thx